tahowallet/extension
# Wallet Connect QA

## What to test

- [ ] Check Injected UI in Wallet Connect modal desktop options
- Connect to dApp
  - [ ] Check dApp information in confirmation prompt
  - [ ] Connect to dApp through Wallet Connect v1
    - Sign a transaction
      - [ ] Reject
      - [ ] Sign
    - [ ] Sign personal message
    - [ ] Sign typed data (e.g. uniswap permit)
  - [ ] Connect to dApp through Wallet Connect v2
    - Sign a transaction
      - [ ] Reject
      - [ ] Sign
    - [ ] Sign personal message
    - [ ] Sign typed data
- Check Top Menu
  - [ ] Check connection guidelines for Wallet Connect
  - [ ] Check new connected status icon

Latest build: [extension-builds-3005](https://github.com/tallyhowallet/extension/suites/10842820585/artifacts/546253828) (as of Wed, 08 Feb 2023 00:38:35 GMT).
